config/config_test.go

```diff
@@ -611,6 +612,7 @@ func TestFullConfig(t *testing.T) {
 	cmpBools(t, "account_required", cfg.AccountRequired, true)
 	cmpBools(t, "auto_gen_source_tid", cfg.AutoGenSourceTID, false)
 	cmpBools(t, "account_adapter_details", cfg.Metrics.Disabled.AccountAdapterDetails, true)
+	cmpBools(t, "account_debug", cfg.Metrics.Disabled.AccountDebug, true)
```

Comment:
For the `TestFullConfig` I would suggest setting the value for `account_debug` to be `false` (different from the default value) in the above `fullConfig` value and assert it here to be `false` as well. This just helps us test through both the flow with default values and non-default values.